test/restore.bats

```diff
@@ -229,50 +144,29 @@ function teardown() {
 	"$CONTAINER_RUNTIME" --root "$RUNTIME_ROOT" delete -f "$ctr_id"
 
 	start_crio
-	run crictl pods --quiet
-	echo "$output"
-	[ "$status" -eq 0 ]
-	[[ "${output}" != "" ]]
+	output=$(crictl pods --quiet)
 	[[ "${output}" == *"${pod_id}"* ]]
 
-	run crictl inspectp "$pod_id"
-	echo "$output"
-	[ "$status" -eq 0 ]
+	output=$(crictl inspectp "$pod_id")
 	[[ "${output}" == *"SANDBOX_NOTREADY"* ]]
 
-	run crictl ps --quiet --all
-	echo "$output"
-	[ "$status" -eq 0 ]
-	[[ "${output}" != "" ]]
+	output=$(crictl ps --quiet --all)
```

Comment:
non-empty output check is removed but it's OK since we have a check for a concrete value below.